deploifai/api/__init__.py

```diff
@@ -298,6 +298,44 @@ def get_cloud_profiles(self, workspace=None) -> typing.List[CloudProfile]:
             ]
         return cloud_profiles
 
+    def create_cloud_profile(self, provider, name, credentials, workspace, fragment):
+        mutation = """
+            mutation(
+              $whereAccount: AccountWhereUniqueInput!
+              $data: CloudProfileCreateInput!
+            ) {
+              createCloudProfile(whereAccount: $whereAccount, data: $data) {
+                ...cloud_profile
+              }
+            }
+        """
+
+        variables = {
+            "whereAccount": {"username": workspace["username"]},
+            "data": {
+                "name": name,
+                "provider": provider.value,
+                provider.value.lower() + "Credentials": credentials,
```

Comment:
Should avoid this code pattern because it tends to lead to buggy behavior in the future. Use conditionals on the enum instead, and explicitly define `awsCredentials` key

Reply:
got it